Makefile.include

```diff
@@ -158,6 +158,20 @@ else
 
 all: link
 
+# Include Board and CPU configuration, if BOARD is not found in BOARDSDIR
+# e.g. when set by the environment fallback to searching in RIOTBOARD
+ifeq (,$(wildcard $(BOARDSDIR)/$(BOARD)/.))
+  ifneq ($(RIOTBOARD),$(BOARDSDIR))
+    # The specified board $(BOARD) was not found in $(BOARDSDIR) fallback to RIOTBOARD)
+    BOARDSDIR = $(RIOTBOARD)
+  endif
```

Comment:
This was on purpose. If the first statement is fulfilled the second one will never be evaluated if `else` is used, but I want it to be evaluated in any case since `BOARDSDIR` value will have changed to `RIOTBOARD`, I can think of other ways of evaluating the conditional but it always ends in evaluating something twice. I could change it to:

```
ifeq (,$(wildcard $(BOARDSDIR)/$(BOARD)/.))
  ifneq ($(RIOTBOARD),$(BOARDSDIR))
    ifeq (,$(wildcard $(RIOTBOARD)/$(BOARD)/.))
      $(error The specified board $(BOARD) is not in RIOTBOARD or BOARDSDIR)
    else
      # The specified board $(BOARD) was not found in $(BOARDSDIR) but in $(RIOTBOARD)
      BOARDSDIR = $(RIOTBOARD)
    endif
  else
    $(error The specified board $(BOARD) is not in RIOTBOARD)
  endif
endif

```

or something like that if you prefer

Reply:
hmm, ok I see.